collect_app/src/test/java/org/odk/collect/android/http/mock/MockHttpInterfaceModule.java

```diff
@@ -0,0 +1,28 @@
+package org.odk.collect.android.http.mock;
+
+import org.odk.collect.android.http.TestableCollectServerClient;
+
+import javax.inject.Singleton;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module
+public class MockHttpInterfaceModule {
+
+    @Provides
+    @Singleton
+    public MockHttpClientConnection provideHttpInterface() {
+        return new MockHttpClientConnection();
+    }
+
+    @Provides
+    @Singleton
+    public TestableCollectServerClient provideCollectServerClient(MockHttpClientConnection httpInterface) {
+        return new TestableCollectServerClient(httpInterface);
+    }
+
+}
+
```

Comment:
Stray lines.

Reply:
Thanks for the review comments @jd-alexander - I'll get those sorted.. looks like I need to watch those carriage returns!

Reply:
Sure no problem! They are just style issues so nothing major!

Reply:
All stray lines should now be removed, thanks to https://github.com/opendatakit/collect/pull/2450 - Thanks @grzesiek2010, very helpful!